dotnet-etcd/multiplexer/Balancer.cs

```diff
@@ -71,15 +72,27 @@ internal class Balancer
                     channel = GrpcChannel.ForAddress(node, options);
                 }
 
+                CallInvoker callInvoker;
+
+                if (interceptors?.Length > 0)
```

Comment:
Although, logically this would work but ideally its not the right way to use `?` operator. As `null` should never be compared with a number (==0, <0, >0 will always return false).